app/utils.py

```diff
@@ -898,6 +901,21 @@ def __init__(self, request, response, env):
         if self.auth and not self.auth.is_valid:
             self.auth = None
 
+        # check admin instead of "login: admin" in app.yaml
+        # because if we use it, user can't sign out when user login as non-admin.
```

Comment:
Please explain why the user cannot sign out i.e., the error page of "login: admin" doesn't have sign-out link.